infra/templates/az-hello-world/backend.tf

```diff
@@ -4,3 +4,8 @@ terraform {
   }
 }
 
+provider "azurerm" {
```

Comment:
Do we need to change these? I thought we pulled the provider in from `/infra/modules/providers/azure/provider`?

Reply:
for a reason, I didn't discover it yet, we have to add the provider in each backend.tf with the features {} expression. Otherwise, we'll get an error

Reply:
Ugh, ok. No need to change, but that is too bad!

Reply:
it is indeed!